### DISCUSSION TITLE
Is this project open for contributions?

### DISCUSSION THREAD

**Discussion by ramarivera** [cooklang/cooklang-obsidian#31] [General]
As the title says :) 

I noticed last commits are from a little over a year ago, but I was wondering if the author was accepting contributions to this plugin?

**Comment by deathau** [comment]
Yeah, no problems!
I've been "away" from my personal projects like this for a while, but I'm starting to get back into them now. There are things higher than this plugin on my list _right now_, but I'm willing to have a look into whatever anyone wants to contribute
